Automap.cs

```diff
@@ -147,17 +259,49 @@ public void Dispose()
                     }
                 }
             }
-            catch (Exception ex) // TODO: Handle file access exceptions
+            catch (IOException ex)
             {
                 // couldn't read from the file
                 Trace("Automap: Error reading line in file.\nError message: " + ex.Message);
+                MessageBox.Show("Error opening transcript file:\n" + ex.Message + "\n\nAutomapping halted.", "File Error", MessageBoxButtons.OK, MessageBoxIcon.Error);   

```

Comment:
Instead of updating the status line and informing the user that the code will continuously try to update, it'll now pop up a standard error message box with (hopefully) useful data, e.g.
![image](https://cloud.githubusercontent.com/assets/4655914/7329065/40d43fe0-ea90-11e4-92db-3767a869798d.png)